src/quo2/components/drawers/permission_context.cljs

```diff
@@ -0,0 +1,24 @@
+(ns quo2.components.drawers.permission-context
+  (:require [react-native.core :as rn]
+            [quo2.foundations.colors :as colors]))
+
+(def radius 20)
+
+(defn permission-context
+  [children]
+  [rn/view
+   {:style {:padding-top             16
```

Comment:
if possible we have started using view and style files.
e.g the sctructure should be 

`src/quo2/components/drawers/permission_context.cljs/view.cljs`

and 
`src/quo2/components/drawers/permission_context.cljs/stylecljs`
these styles should be imported into the view file then.